starlette/websockets.py

```diff
@@ -22,6 +22,14 @@ def __init__(self, code: int = 1000, reason: str | None = None) -> None:
         self.reason = reason or ""
 
 
+class WebSocketDisconnected(RuntimeError):
+    """
+    Raised when attempting to use a disconnected WebSocket.
+    """
+
+    pass
```

Comment:
That's not really what I meant... The exception already exists.

Reply:
Well, reading the code, `WebSocketDisconnect` seems to be called when you want to disconnect a socket, in these cases the socket is already disconnected. In my opinion, it would make more sense to create a new exception (`WebSocketDisconnected`). Tell me if you still want to use `WebSocketDisconnect` and if so I will make such changes.